src/dbnode/storage/bootstrap/types.go

```diff
@@ -90,13 +100,11 @@ type ProcessOptions interface {
 
 // RunOptions is a set of options for a bootstrap run.
 type RunOptions interface {
-	// SetIncremental sets whether this bootstrap should be an incremental
-	// that saves intermediate results to durable storage or not.
-	SetIncremental(value bool) RunOptions
+	// SetIncrementalConfig sets incremental configuration for this bootstrap.
+	SetIncrementalConfig(value IncrementalConfig) RunOptions
 
-	// Incremental returns whether this bootstrap should be an incremental
-	// that saves intermediate results to durable storage or not.
-	Incremental() bool
+	// IncrementalConfig returns the incremental configuration for this bootstrap.
+	IncrementalConfig() IncrementalConfig
```

Comment:
maybe time to rename this guy to `PersistConfig` too (talked to @robskillington, that's the best we could come up with)